ci-operator/step-registry/dpf/hypervisor/networking-tests/dpf-hypervisor-network-tests-commands.sh

```
@@ -0,0 +1,140 @@
+
+else 
+  echo "DPF spot checks tests Failed"; 
+  exit 1
+fi
+
+
+
+echo "=== Run DPF Kubernetes Traffic Flow Tests on Existing Cluster ==="
+# Run kubernetes traffic flow test
+# Need to run cmds or script on hypervisor to discover worker node names after being renamed
+if ssh ${SSH_OPTS} root@${REMOTE_HOST} "ls -ltr; \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)